RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -132,7 +143,20 @@
 trackingPhase1QuadProp.toModify(mixedTripletStepSeedLayersB, layerList = ['BPix3+BPix4+TIB1'])
 
 # TrackingRegion
-mixedTripletStepTrackingRegionsB = mixedTripletStepTrackingRegionsA.clone(RegionPSet = dict(ptMin=0.6, originHalfLength=10.0))
+mixedTripletStepTrackingRegionsB = mixedTripletStepTrackingRegionsA.clone(RegionPSet = dict(ptMin=0.6))
+
+if not pp_on_XeXe_2017.isChosen():  
+    mixedTripletStepTrackingRegionsB.RegionPSet.originHalfLength = 10.0
+
```

Comment:
Why didn't my proposal work? (ok, I admit it was a bit incorrect, it should have been
```python
_mixedTripletStepTrackingRegionsB = mixedTripletStepTrackingRegionsA.clone(RegionPSet = dict(ptMin=0.6))
mixedTripletStepTrackingRegionsB = _mixedTripletStepTrackingRegionsB.clone(RegionPSet = dict(originHalfLength=10.0))
pp_on_XeXe_2017.toReplaceWith(mixedTripletStepTrackingRegionsB, _mixedTripletStepTrackingRegionsB)
```
i.e. with `RegionPSet` in the second `clone()` )

Reply:
The error is this one, which persists with your latest suggestion.  I haven't had time to figure out how to fix it.

python encountered the error: <type 'exceptions.TypeError'>
originHalfLength does not already exist, so it can only be set to a CMS python configuration type